tests/bats/tests.bats

```diff
@@ -41,15 +41,17 @@ setup() {
 }
 
 @test "cwd spt" {
-  run ${TIMEOUT} --foreground 30s ${SOLO5_SPT} cwd_test.bin '{"cmdline":"cwd_test /etc","cwd":"/etc"}'
+  dd if=/dev/zero of=dummy bs=512 count=1 > /dev/null 2>&1
+  create_tap
+  run ${TIMEOUT} --foreground 30s ${SOLO5_SPT} --block:rootfs=dummy --net:tap=${NET} cwd_test.bin '{"cmdline":"cwd_test /etc","cwd":"/etc"}'
   [ "$status" -eq 0 ]
   [[ "$output" == *"=== main() of \"cwd_test\" returned 0 ==="* ]]
 }
 
 @test "cwd hvt" {
-  touch dummy
+  dd if=/dev/zero of=dummy bs=512 count=1 > /dev/null 2>&1
```

Comment:
just curious, do tests fail with an empty file (dummy's size of 0)?

Reply:
The spt tender says:
```
solo5-spt: dummy: Backing storage must be at least 1 block (512 bytes) in size
```

Reply:
oh, got it